cloud.foundry.cli/src/test/java/cloud/foundry/cli/operations/ApplicationsOperationsTest.java

```diff
@@ -223,6 +216,47 @@ public void testRemoveApplicationShouldThrowNullPointerExceptionWhenApplicationN
         assertThrows(NullPointerException.class, () -> applicationsOperations.remove(null));
     }
 
+    @Test
+    public void testRenameOnCurrentNameNullThrowsNullPointerException() {
+        //given
+        ApplicationsOperations applicationsOperations = new ApplicationsOperations(
+                mock(DefaultCloudFoundryOperations.class));
+
+        //when
+        assertThrows(NullPointerException.class, () ->
+                applicationsOperations.rename("appName", null));
+    }
+
+    @Test
+    public void testRenameOnNewNameNullThrowsNullPointerException() {
+        //given
+        ApplicationsOperations applicationsOperations = new ApplicationsOperations(
+                mock(DefaultCloudFoundryOperations.class));
+
+        //when
+        assertThrows(NullPointerException.class, () ->
+                applicationsOperations.rename("appName", null));
+    }
+
+    @Test
+    public void testRename() {
```

Comment:
you should describe your test name more precise. 
This test-method is the "happy-path". 
What is the situation - what action did you performed - what should be the expected output/result?

Reply:
changed